pkg/chaosdaemon/ipset_server.go

```diff
@@ -30,6 +30,9 @@ const (
 )
 
 func (s *Server) FlushIpSet(ctx context.Context, req *pb.IpSetRequest) (*empty.Empty, error) {
+	if ctx.Err() != nil{
```

Comment:
can we check context Error before? 
or use gRPC interceptor feature?

Reply:
Good idea. Working on this via interceptor.